src/bud/api/auto.ts

```diff
@@ -23,9 +23,4 @@ const auto: Auto = function (options: {
 }
 
 export {auto}
-
-import type {bud} from '../'
-
-export type Auto = (options: {
-  [key: string]: [string]
-}) => bud
+import {Bud, Auto} from '.'
```

Comment:
Ok, now I am a bit more opinionated here (compared to the type import) in thinking that regular imports should proceed exports and any other code. It seems a bit non-standard even for other languages.

Reply:
totally agree. was trying to keep the types separate from the scripts, as mentioned in our convo about it. fixed.